src/com/jalasoft/search_files/utils/validation.py

```diff
@@ -1,35 +1,56 @@
 import os
-
+from src.com.jalasoft.search_files.utils.logging_config import logger
 
 class Validation(object):
     def __init__(self):
         pass
 
     def is_number(self, cadena):
-        response_bool = cadena.isdigit()
-        return response_bool
+        logger.info("Starting the method")
+        boolean_contain = cadena.isdigit()
+        logger.info("The value return is: %s", boolean_contain)
+        logger.info("Exit method")
+        return boolean_contain
 
     def is_the_range(self, number, upper_limit, lower_limit):
+        logger.info("Starting the method")
         number = int(number)
         if number >= lower_limit and number <= upper_limit:
-            return True
+            boolean_contain = True
         else:
-            return False
+            boolean_contain = False
+        logger.info("The value return is: %s", boolean_contain)
+        logger.info("Exit method")
+        return boolean_contain
 
     def attempt_maximum(self, intentos):
+        logger.info("Starting the method")
         intentos = int(intentos)
         if intentos <= 5:
-            return True
+            boolean_contain = True
         else:
-            return False
+            boolean_contain = False
+        logger.info("The value return is: %s", boolean_contain)
```

Comment:
please change the meesage, it could be:
logger.info("The value returned is: %s", boolean_contain)